packages/angular/projects/clr-angular/src/data/datagrid/datagrid-column.ts

```diff
@@ -39,14 +39,15 @@ import { ClrPopoverPositionService } from '../../utils/popover/providers/popover
 import { ClrPopoverEventsService } from '../../utils/popover/providers/popover-events.service';
 import { ClrPopoverToggleService } from '../../utils/popover/providers/popover-toggle.service';
 import { DetailService } from './providers/detail.service';
+import { Directions } from '@clr/core/internal';
```

Comment:
@gnomeontherun Need a special review for this file.

It brings a little breaking change in case someone extends the `sort` function and changes the `shape` icon of the sorting. I hope nobody does that but it is a case.